src/frontend/src/pages/home/home.tsx

```diff
@@ -1,7 +1,118 @@
 import React from 'react';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import { Link, } from "react-router-dom";
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Box from '@material-ui/core/Box';
+import Paper from '@material-ui/core/Paper';
+
 
 export const Home: React.FC = () => {
     return (
-        <div>Home Page</div>
+        <Box>
+
+            {/*     navigation bar      */}
+
+            <AppBar position="static">
+                <Toolbar>
+                    <Grid
+                        justify="space-between"
+                        container
```

Comment:
```suggestion
                        alignItems="center"
                        container
```
Just to make sure everything is aligned in the vertical direction to the center. The score in the centre being off centre horizontally is unfortunate, but I think it should do for now.